actions/ostree_deploy_action.go

```diff
@@ -93,18 +93,21 @@ func (ot *OstreeDeployAction) setupFSTab(deployment *ostree.Deployment, context
 
 func (ot *OstreeDeployAction) Run(context *debos.DebosContext) error {
 	ot.LogStart()
-	/* First deploy the current rootdir to the image so it can seed e.g.
-	 * bootloader configuration */
-	err := debos.Command{}.Run("Deploy to image", "cp", "-a", context.Rootdir+"/.", context.ImageMntDir)
-	if err != nil {
-		return fmt.Errorf("rootfs deploy failed: %v", err)
+
+	if _, err := os.Stat(context.ImageMntDir); err == nil {
```

Comment:
Could use a comment explaining this is to handle cases where we didn't partition an image. Do we need to run os.Stat here though rather then simply checking if ImageMntDir is set (e.g. non-""), If ImageMntDir is set but doesn't exist we have bigger problems tbh

Reply:
done